store/tikv/2pc.go

```diff
@@ -771,6 +805,20 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 	return nil
 }
 
+func (c *twoPhaseCommitter) doActionWithRetry(ctx context.Context, f func(ctx context.Context) error) (err error) {
```

Comment:
Here use `doAction` instead of `f` will be easier to understand for me.

Reply:
I think `with retry` is better...